File: LottieLib_test/app/src/main/java/com/daeseong/lottielib_test/Main8Activity.java
package com.daeseong.lottielib_test;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.webkit.WebView;
import android.webkit.WebViewClient;

public class Main8Activity extends AppCompatActivity {

    private static final String TAG = Main8Activity.class.getSimpleName();

    private WebView wbv;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main8);

        wbv = (WebView)findViewById(R.id.wbv);
        wbv.getSettings().setJavaScriptEnabled(true);
        wbv.setWebViewClient(new WebViewClient());
        wbv.loadUrl("file:///android_asset/test.html");
    }
}

File: lottielib_test_kotlin/app/src/main/java/com/daeseong/lottielib_test/Main8Activity.kt
package com.daeseong.lottielib_test

import android.os.Bundle
import android.webkit.WebView
import android.webkit.WebViewClient
import androidx.appcompat.app.AppCompatActivity

class Main8Activity : AppCompatActivity() {

    private val tag = Main8Activity::class.java.simpleName

    private lateinit var wbv: WebView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main8)

        wbv = findViewById(R.id.wbv)
        wbv.settings.javaScriptEnabled = true
        wbv.webViewClient = WebViewClient()
        wbv.loadUrl("file:///android_asset/test.html")
    }
}